identity-apps-core/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties

```diff
@@ -97,6 +97,7 @@ select.tenant.dropdown.display.name=Select Tenant
 super.tenant.display.name=Super Tenant
 super.tenant=carbon.super
 domain.unknown=domain.unknown
+idp.not.assigned.to.sp=idp.not.assigned.to.sp
```

Comment:
Is using key for the value here intentional? 

Check the other places as well.